src/main/java/io/tarantool/driver/clientbuilder/TarantoolProxyClientBuilder.java

```diff
@@ -0,0 +1,17 @@
+package io.tarantool.driver.clientbuilder;
+
+import io.tarantool.driver.ProxyTarantoolTupleClient;
+import io.tarantool.driver.api.TarantoolClient;
+import io.tarantool.driver.api.TarantoolResult;
+import io.tarantool.driver.api.tuple.TarantoolTuple;
+import io.tarantool.driver.proxy.ProxyOperationsMappingConfig;
+
+public interface TarantoolProxyClientBuilder
+        extends TarantoolDecoratedClientBuilder
+        <ProxyTarantoolTupleClient, TarantoolProxyClientBuilder,
+                TarantoolClient<TarantoolTuple, TarantoolResult<TarantoolTuple>>> {
```

Comment:
Maybe that's more readable? (for all big generics)
```suggestion
<ProxyTarantoolTupleClient,
 TarantoolProxyClientBuilder,
 TarantoolClient<TarantoolTuple, TarantoolResult<TarantoolTuple>>> {
```